MapboxNavigation/Resources/Base.lproj/Navigation.storyboard

```diff
@@ -11,6 +11,11 @@
         <capability name="Unknown constraint types" minToolsVersion="5.1"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
+    <customFonts key="customFonts">
```

Comment:
Are we using this font? The style currently doesn't support changing font family if we use more than one font.

Reply:
- [ ] No custom fonts